cachito/workers/utils.py

```diff
@@ -19,25 +24,42 @@ def extract_app_src(archive_path, parent_dir):
     return os.path.join(parent_dir, 'app')
 
 
-def get_request_bundle_dir(request_id):
+class RequestBundleDir(paths.RequestBundleDir):
```

Comment:
It might be nice to have a separate Python module called `paths.py` in `cachito/workers` as well. I'd like to avoid `utils.py` becoming a dumping ground when possible.